src/renderer/services/thorchain/thorNode.ts

```diff
@@ -0,0 +1,86 @@
+import * as RD from '@devexperts/remote-data-ts'
+import { Address } from '@xchainjs/xchain-client'
+import { baseAmount } from '@xchainjs/xchain-util'
+import * as FP from 'fp-ts/function'
+import * as Rx from 'rxjs'
+import * as RxAjax from 'rxjs/ajax'
+import * as RxOp from 'rxjs/operators'
+
+import { Network } from '../../../shared/api/types'
+import { envOrDefault } from '../../helpers/envHelper'
+import { liveData } from '../../helpers/rx/liveData'
+import { ErrorId } from '../wallet/types'
+import { NodeInfoLD, NodeStatus, ThorNodeApiUrlLD } from './types'
+
+// Tmp type as ThorNodeApi does not provide valid swagger spec yet
+// TODO remove after https://github.com/thorchain/asgardex-electron/issues/763
+type NodeInfoResponse = {
+  status: string
+  bond: string
+  current_award: string
+}
+
+const getNodeStatusByString = (stringStatus: string): NodeStatus => {
+  const lowerCasedStatus = stringStatus.toLowerCase()
+
+  switch (lowerCasedStatus) {
+    case 'active': {
+      return 'active'
+    }
+    case 'standby': {
+      return 'standby'
+    }
+    case 'disabled': {
+      return 'disabled'
+    }
+    default: {
+      return 'unknown'
+    }
+  }
+}
+
+const TESTNET_THORNODE_API = envOrDefault(process.env.REACT_APP_TESTNET_THORNODE_API, '')
+
+const thorNodeApiAddress$ = (network: Network): ThorNodeApiUrlLD => {
+  if (!!TESTNET_THORNODE_API && network === 'testnet') {
+    return Rx.of(RD.success(TESTNET_THORNODE_API))
+  }
+  return Rx.of(RD.failure({ errorId: ErrorId.GET_THORNODE_API, msg: 'Not implemented yet' }))
+}
+
+const getNodeInfo$ = (node: Address, network: Network): NodeInfoLD =>
+  FP.pipe(
+    thorNodeApiAddress$(network),
+    liveData.chain((thorApi) =>
+      FP.pipe(
+        RxAjax.ajax(`${thorApi}/node/${node}`),
+        RxOp.map(({ response }) => response),
+        RxOp.map(RD.success),
+        RxOp.catchError(() =>
+          Rx.of(
+            RD.failure({
+              errorId: ErrorId.GET_NODE,
+              msg: `Failed to load info for ${node}`
+            })
+          )
+        )
+      )
+    ),
+    liveData.map((nodeData: NodeInfoResponse) => ({
+      bond: baseAmount(nodeData.bond),
+      award: baseAmount(nodeData.current_award),
```

Comment:
[nitpick] ^ `baseAmount(..., THOR_DECIMAL)`

Reply:
sure. will add. but 8 is a default value anyway :)